### PR TITLE
Summarize Notebook

### PR DESCRIPTION
**Pull request by leilacc**
Still need to update links to model package

### CODE REVIEW

File: notebooks/Deploy summarize.ipynb [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Mapping for Model Packages (initially only us-east-1 and eu-west-1 is supported)\n",
+    "model_package_map = {\n",
```
Comment (jpekmez) [inline notebooks/Deploy summarize.ipynb:86]: can you add all regions please? example: https://github.com/cohere-ai/cohere-sagemaker/blob/945dce624e5c809c3b8788a5e729dc9ac06eb796/notebooks/Deploy%20command%20XL.ipynb

Comment (leilacc) [reply notebooks/Deploy summarize.ipynb:86]: How do we decide which regions to allow?

Comment (jpekmez) [reply notebooks/Deploy summarize.ipynb:86]: we want all :)

**Review by jpekmez — APPROVED** [review]

File: notebooks/Deploy summarize.ipynb [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
```
Comment (jpekmez) [inline notebooks/Deploy summarize.ipynb:86]: we need to add the `cohere_package` when it's published

Comment (leilacc) [reply notebooks/Deploy summarize.ipynb:86]: will do

**Pull request merged**